kymatio/scattering1d/tests/test_numpy_scattering1d.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
These tests don't cover the whole set of functionality (like the torch tests and the 2D NP tests). This is fine, for now, but we should make an issue and fix this.

Reply:
I agree.